wafer/users/views.py

```diff
@@ -37,9 +38,11 @@ class UsersView(ListView):
     paginate_by = 25
 
     def get_queryset(self, *args, **kwargs):
+        qs = super(UsersView, self).get_queryset(*args, **kwargs)
         if not settings.WAFER_PUBLIC_ATTENDEE_LIST:
-            raise Http404()
-        return super(UsersView, self).get_queryset(*args, **kwargs)
+            qs = qs.filter(talks__status=ACCEPTED).distinct()
```

Comment:
Otherwise the speaker appears multiple times if they have multiple talks. I noticed this with my sample dc16 dataset.